tests/integration/throttle.go

```diff
@@ -369,8 +375,14 @@ func (s *CCVTestSuite) TestDoubleSignDoesNotAffectThrottling() {
 	// Recv 500 packets from consumer to provider in same block
 	for _, packet := range packets {
 		consumerPacketData := ccvtypes.ConsumerPacketData{}
-		ccvtypes.ModuleCdc.MustUnmarshalJSON(packet.GetData(), &consumerPacketData)
-		providerKeeper.OnRecvSlashPacket(s.providerCtx(), packet, *consumerPacketData.GetSlashPacketData())
+		consumerPacketDataV1 := ccvtypes.ConsumerPacketDataV1{}
```

Comment:
All tests were changed to support V1 data packets. The consumer will `always` send packets in V1 format over the wire